src/python/WMCore/MicroService/Unified/MSOutput.py

```diff
@@ -301,11 +288,8 @@ def makeSubscriptions(self, workflow):
 
                 # overwrite the current DiskDestination by something that will work for DDM
```

Comment:
So unlike Unified, the output disk site choice is resolved here? Because as I understand, Unified just forwards the request to dynamo (with a generic or `null` list of sites) in https://github.com/CMSCompOps/WmAgentScripts/blob/master/Unified/closor.py#L585 (perhaps @dr-stringfellow can confirm) If so, we should review how the disk site is chosen.

Reply:
Actually, the disk destination is decided in this block:
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/MicroService/Unified/MSOutput.py#L755-L791
but noticed that the destination is already in the format of an RSE Expression (Rucio compliant). 

In this section of the code, we are making data transfers against DDM, so we need to massage a bit the list of sites (for RelVals, it does not change). For standard workflows, we simply pass T1 Disk and T2s, and leave the final decision to DDM.